### PR TITLE
Added support for exec, stats, and logs --since

### PR DESCRIPTION
**Pull request by stuclem**
Updated supported Docker commands table to state support for `exec` and `stats`.

@chengwang86 and @anchal-agrawal can you please review?

### CODE REVIEW

**Comment by stuclem** [comment]
Thanks @chengwang86 !

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -57,7 +57,7 @@
...
 |`start`|[Start a container](https://docs.docker.com/engine/reference/commandline/start/)|Yes, since 1.0. Supports the `--attach` and `--interactive` options.|
-|`stats`|[Get container stats based on resource usage](https://docs.docker.com/engine/reference/commandline/stats/)|Yes, since 1.1. Provides statistics about CPU and memory usage. Does not yet provide statistics about network or disk usage.|
+|`stats`|[Get container stats based on resource usage](https://docs.docker.com/engine/reference/commandline/stats/)|Yes. Provides statistics about CPU and memory usage since 1.1. Provide statistics about network or disk usage since 1.2.|
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/container_operations.md]: `Provide` -> `Provides`?

Comment (stuclem) [inline docs/user_doc/vic_app_dev/container_operations.md]: Good spot. @anchal-agrawal. Fixed!

**Comment by stuclem** [comment]
So, if there are no caveats about exec or stats, I think that we're good to go. Thanks @anchal-agrawal and @chengwang86 !

Fixes https://github.com/vmware/vic-product/issues/326
Fixes https://github.com/vmware/vic-product/issues/79

**Comment by stuclem** [comment]
Added support for `docker-logs --since`. Fixes https://github.com/vmware/vic-product/issues/78.

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 2]
```diff
@@ -45,10 +45,10 @@
...
 |`events`|[Get real time events from the server](https://docs.docker.com/engine/reference/commandline/events/)|Yes, since 1.0. Supports passive Docker events for containers and images. Does not yet support events for volumes or networks.|
-|`exec`|[Run a command in a running container](https://docs.docker.com/engine/reference/commandline/exec/)|No|
+|`exec`|[Run a command in a running container](https://docs.docker.com/engine/reference/commandline/exec/)|Yes, since 1.2|
 |`export`|[Export a container](https://docs.docker.com/engine/reference/commandline/export/)|No|
 |`kill`|[Kill a running container](https://docs.docker.com/engine/reference/commandline/kill/)|Yes, since 1.0. Docker must wait for the container to shut down.|
-|`logs`|[Get container logs](https://docs.docker.com/engine/reference/commandline/logs/)|Yes, since 1.0. Does not support `docker logs --timestamps` (`-t`) and `--since` options.|
+|`logs`|[Get container logs](https://docs.docker.com/engine/reference/commandline/logs/)|Yes, since 1.0. Supports `--since` since 1.2. Does not support `--timestamps` (`-t`). |
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/container_operations.md]: `logs --timestamps` is supported (https://github.com/vmware/vic/pull/4209) - pinging @jzt for review.

Comment (jzt) [inline docs/user_doc/vic_app_dev/container_operations.md]: `--since` and `--timestamps` confirmed

**Comment by stuclem** [comment]
Thanks @jzt, `--timestamps` is now listed as supported.

**Comment by stuclem** [comment]
Thanks @jzt and @anchal-agrawal !

**Pull request merged**